### DISCUSSION TITLE
how to get detailed info of error: Weird server reply

### DISCUSSION THREAD

**Discussion by peguine** [curl/curl#13723] [Q&A]
I am using libcurl to download a picture , the error message is "Weird server reply ". but when i try again , it may return ok, How to locate the problem?

**Comment by peguine** [comment]
i was using  libcurl 7.81 before ,everything is ok ,but the website updated to http3, so i update libcurl 8.7.1, then it  have this error  "Weird server reply "

**Comment by icing** [comment]
@peguine if you upgraded to a more recent curl, you could try to reproduce the problem via the `curl` command line, using args like `-v --trace-config time,ids,write` to see more what is going on. When this only happens in your libcurl application, you might want to set VERBOSE and a DEBUGFUNCTION and collect the output yourself in a file for analysis.

Somehow, we'd need to actual trace of the data sent and received. Otherwise we are unable to help, most likely.
